proposals/lip-0048.md

```diff
@@ -25,15 +25,59 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
+
+The option to transfer the consumed fees to a dedicated pool address `ADDRESS_FEE_POOL` could be beneficial for sidechains, as fees collected this way can be used as an incentive for validators and other protocol participants for various purposes: Funding a community pool for on-chain governance spending, sharing funds proportionally among validators per round, incentivizing ecosystem developers, supporting chain maintenance and improvement, rewarding users' staking and voting activities, etc.
+
+In the Lisk mainchain, the consumed fees are always burned. This reduces the total supply, creating a deflationary pressure that works against inflation.
+
+### Cross-chain Message Fees
+
+Cross-chain messages (CCMs) facilitate interoperability by enabling cross-chain transfer of data. Each CCM contains a [*message fee*][lip-0049#ccmschema], intended to cover the cost of transaction processing on the receiving chain (see [LIP 0049][lip-0049#fee]). 
+
+Since managing message fees requires checking and updating the escrow balances for the message fee token, this task is assigned to the Token module. In particular, paying for message fees is called for in the Interoperability module, whose [`sendInternal`][lip-0045#sendinternal] method uses the [`payMessageFee`][lip-0051#paymessagefee] of the Token module. After the Token module has ensured that the message fee has been credited to the relayer's account, the Fee module handles the transaction fee processing in the receiving chain.
+
+For protocol reasons (see, e.g., [`bounce`][lip-0045#bounce] and [`terminateChain`][lip-0045#terminatechain] methods in LIP 0045), the Interoperability module allows the message fee to be zero. Consequently, the choice of setting a minimum message fee is delegated to each module that provides its own cross-chain commands. Nevertheless, there is a minimum fee in case the user wants the failed CCM to be returned (see [`bounce`][lip-0045#bounce] method in LIP 0045). Thus, even though this fee is not mandatory, it is advisable to treat it as a suggested minimum fee. 
+
+Furthermore, the CCM sender should ensure to include any additional fees necessary for the CCM execution, e.g., to pay for the user account initialization on the receiving chain. 
+
+Similar to transaction fees, the consumed part of the message fee, which might only be additional fees (as there is no mandatory minimum fee), can either be burned or transferred to a fee pool, depending on the chain configuration. The available part of the message fee is left to the relayer to cover the cost of CCM execution and to incentivize its participation in CCM relaying. However, in contrast to including transactions in a block, a relayer cannot prioritize CCMs with higher fees as the interoperability protocol requires them to be sent in order. This also implies that users have no reason to pay higher message fees than are required for successful CCM execution.
```

Comment:
```suggestion
The fee processing is similar to transaction fees; note that the consumed part of the message fee contains only the so-called additional fees,  as there is no mandatory minimum fee. Similarly to transaction fees, the consumed part can either be burned or transferred to a fee pool, depending on the chain configuration. The available part of the message fee is left to the relayer to cover the cost of CCU execution (note the relayer pays transaction fees for posting the CCU in the receiving chain, since a CCU is a regular transaction) and to incentivize its participation in CCM relaying. However, in contrast to including transactions in a block, a relayer cannot prioritize CCMs with higher fees as the interoperability protocol requires them to be sent in order. This also implies that users have no reason to pay higher message fees than are required for successful CCM execution.
```

Reply:
Also in the very end, we could even say that (because users no need to pay more fee to include their ccm) a consequence of this is that users know that the ccms would be delivered as soon as a relayer decides to post a ccm in the target chain, avoiding this way extra delays on delivering the ccm, something like that.